downloads.md

```diff
@@ -15,56 +15,76 @@ title: Downloads
 <div class="col-xs-12">
     <ul class="nav nav-tabs nav-justified" role="tablist">
         <li role="presentation" class="active">
-            <a href="#system1" aria-controls="system1" role="tab" data-toggle="tab"><img src="img/redhat.png" width="20"><span class="myHidden-sm">&nbsp;Fedora/CentOS</span></a>
+            <a href="#system1" aria-controls="system1" role="tab" data-toggle="tab"><img src="img/docker.png" width="20"><span class="myHidden-sm">&nbsp;Docker</span></a>
         </li>
         <li role="presentation">
-            <a href="#system2" aria-controls="system2" role="tab" data-toggle="tab"><img src="img/Ubuntu.png" width="20"><span class="myHidden-sm">&nbsp;Debian/Ubuntu</span></a>
+            <a href="#system2" aria-controls="system2" role="tab" data-toggle="tab"><img src="img/redhat.png" width="20"><span class="myHidden-sm">&nbsp;Enterprise Linux</span></a>
```

Comment:
It might look funny on some screens due to length; perhaps we may better call this `RHEL-alike`